CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Ewon Flexy Extensions Library Changelog
 
+## Version 1.5.0
+### Features
+- Add friendly date/time string utility method
```

Comment:
friendly?

Reply:
Changed description and commit name to be more descriptive